main/modes/system/intro/tutorial.c

```diff
@@ -104,6 +104,7 @@ void tutorialCheckTriggers(tutorialState_t* state)
             state->loudSound = false;
 
             next = state->curStep;
+            printf("next state is %d\n", (int)next);
```

Comment:
I recommend using the `ESP_LOG*()` functions instead of `printf()`:
```suggestion
            ESP_LOGI("TUT", "next state is %p\n", next);
```
This way we can control log level output (i.e. only print errors, not info) and you get a nice decoration in VSCode:
```
I (8416) TUT | main/modes/system/intro/tutorial.c:107 - tutorialCheckTriggers() | next state is 0x5a397d2b1520
```
And even better, the `main/modes/system/intro/tutorial.c:107` part is clickable and will jump to the source!

(also you're trying to print a pointer here)